src/features/stock-ticker/redux/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const UPDATE_STOCK_TICKER = "UPDATE_STOCK_TICKER";
+export type UPDATE_STOCK_TICKER = typeof UPDATE_STOCK_TICKER;
```

Comment:
No need to do this way.
`export const UPDATE_STOCK_TICKER = "UPDATE_STOCK_TICKER"` will automatically do it for you

Reply:
Will change all of my constants in another PR